server/plugins/schema-plugin-ob-oracle/src/main/java/com/oceanbase/odc/plugin/schema/oboracle/parser/OBOracleGetDBTableByParser.java

```diff
@@ -328,8 +328,8 @@ private CreateIndex parseIndexDDL(String ddl) {
         return statement;
     }
 
-    private static String readFile(String strFile) throws IOException {
-        try (InputStream input = new FileInputStream(strFile)) {
+    private String readFile(String strFile) throws IOException {
+        try (InputStream input = OBOracleGetDBTableByParser.class.getClassLoader().getResourceAsStream(strFile)) {
```

Comment:
OBOracleGetDBTableByParser.class.getClassLoader may be replaced by getClass().getLassLoader in instance method

Reply:
👌